.gitlab-ci.yml

```diff
@@ -13,7 +13,7 @@ test-api:
     image: python:3.7-slim
     stage: test
     services:
-        - postgres:10.9-alpine
```

Comment:
Any reason why this was locked to 10.9(apart from the fact that it was the latest version at the time)?

Reply:
We can probably remove the .gitlab-ci file now - what do you think @matthewelwell ?

Reply:
Yep, might as well remove (but probably do it in a separate PR to keep the git history clean)

Reply:
#227